exercises/nth-prime/runtests.jl

```diff
@@ -0,0 +1,27 @@
+using Test
+
+include("nthprime.jl")
+include("example.jl")

```

Comment:
```suggestion
```

`example.jl` is used to test that the `runtests.jl` file works, basically to test the tests. When a student downloads the exercise, they won't download the `example.jl` file so this would error.